volatility3/framework/plugins/windows/pstree.py

```diff
@@ -149,19 +172,24 @@ def run(self):
             else "(P)"
         )
 
+        cols = [
+            ("PID", int),
+            ("PPID", int),
+            ("ImageFileName", str),
+            (f"Offset{offsettype}", format_hints.Hex),
+            ("Threads", int),
+            ("Handles", int),
+            ("SessionId", int),
+            ("Wow64", bool),
+            ("CreateTime", datetime.datetime),
+            ("ExitTime", datetime.datetime),
+        ]
+
+        if self.config["verbose"]:
```

Comment:
I think there are other plugins where we return a different TreeGrid (ie, different result columns) based on a flag, but I'm a little bit nervous about it, it'll make it slightly harder for frameworks to know what to expect.  Again, not a show stopper, but given it doesn't take much more to find the additional values, perhaps we should just always include them from the get-go?

Reply:
@ikelos does adding columns to a TreeGrid for an existing plugin require bumping the plugin interface version? Could clients be affected by receiving unexpected columns without protecting it with a flag?

I agree, making this the default could be a good approach, I just wanted to clarify the ramifications of such a change.

Reply:
To this point, the version number has only been about the internally accessible API (so methods within the class).  We haven't ever told frameworks that plugin output will be fixed for a particular framework (although I know at least one which has written specific handlers for the output of each plugin).  The column names are programatically accessible, so as long as this is additive it should be fine.  We might also want to get in the habit of getting people to bump the patchj version of a plugin any time there is a change (as we should have been doing, but have generally ignored).

We probably need to make a policy call on this at some point, but for now I'm happy with bump or no bump.  If the version number is to be bumped, I'd only do the minor version, because a major version would require all plugins depending on it to have their dependencies updated.

Reply:
Decided to go the route of always including the additional rows in the TreeGrid. Updated with [5fb7178](https://github.com/volatilityfoundation/volatility3/pull/1076/commits/5fb7178811248af97e3569f72c42b847f6069ff5)